### ISSUE TITLE
Task Inspector tweaks

### ISSUE THREAD

**Issue by djmitche** [taskcluster/taskcluster-web#252]
I like the idea of hiding things behind twirldowns (such as the payload now).  But I think the choices of what is displayed by default and what is not are a bit odd.

* Dependencies are a little expensive to fetch, and not often needed -- let's hide those by default
  * Dependencies and Requires are interrelated and should be bundled together
* TaskGroupId isn't displayed..
* Most of the details can be hidden behind a "more.." click:
  * Retries left, Deadline, Expires, Priority, Provisioner, WorkerType, SchedulerId
* Tags appears to show "n/a" even though there are tags.  Those could probably be included in pills with "owned by"
* Scopes and Routes can also be hidden
  * It would be great to display these as list items, rather than preformatted text with overflow=scroll.  Particularly for scopes, usually the right end of the string is the interesting bit, and the scroll just cuts that off.
* The right-hand pane is titled "Task Runs".  It'd be great if that has the run number in it, making it more clear if there is more than one (because it won't say "Run 0")

**Comment by helfi92** [comment]
> Dependencies are a little expensive to fetch, and not often needed -- let's hide those by default

Are you experiencing any slowness?

> TaskGroupId isn't displayed..

Ah, right! I'll add it.

> Tags appears to show "n/a" even though there are tags. Those could probably be included in pills with "owned by"

![screen shot 2018-11-02 at 1 56 44 pm](https://user-images.githubusercontent.com/3766511/47932127-2f57fb80-dea7-11e8-9d4f-a05f828500f7.png)

Do they have to be text selectable?

> The right-hand pane is titled "Task Runs". It'd be great if that has the run number in it, making it more clear if there is more than one (because it won't say "Run 0")

The run number is shown below.

![screen shot 2018-11-02 at 1 58 39 pm](https://user-images.githubusercontent.com/3766511/47932409-f9674700-dea7-11e8-9c12-897864374b98.png)

![screen shot 2018-11-02 at 2 02 49 pm](https://user-images.githubusercontent.com/3766511/47932416-05530900-dea8-11e8-8d7b-a97b74db16c9.png)

If there were multiple runs, there would be multiple dots. The user can switch runs by clicking the "Next" and "Previous" buttons.

**Comment by helfi92** [comment]
@djmitche Let me know how it looks now. https://taskcluster-web.netlify.com/tasks/

**Comment by djmitche** [comment]
> Are you experiencing any slowness?

It's not that they're slow (although they might be), it's that it's putting a load on the queue service that is usually unnecessary.

> Do they have to be text selectable?

I'd prefer that.  Day-to-day use of this tool involves a lot of copy/pasting into irc and bugs.  I don't know how often tags are often copy/pasted, but it would be annoying to find out they can't be when I need to..

> The run number is shown below.

I stared at that screenshot for a while, reading through it three times to find the run number.  I know the dots symbolically represent the run number, but that's hard to parse and unless there are several runs not at all obvious.  Also, the card says "Task Runs" when in fact it only displays one "Task Run".  If you want to know which, you're left counting (and, given the common case of two runs, trying to remember if material design shows the selected dot as brighter or dimmer than the unselected dot).

In the existing UI, it's a common source of confusion to open up a task which Treeherder has marked as failed, and see a success status.  The sheriffs have gotten used to checking the title of one of the tabs midscreen to see what run number it says.  It seems a shame to further hide that information and potentially deepen the confusion.

**Comment by djmitche** [comment]
Oops, I submitted before commenting on the running version.  It's looking a lot better!

![image](https://user-images.githubusercontent.com/28673/48030209-58ce8c80-e11e-11e8-8964-726972d2597d.png)

still shows "Requires" and "Dependencies" as separate.  Requires is really just a characterization of the dependencies, and does not deserve its own "section" of the card.

Provisioner is now behind "More.." but I still see "workerType".  Those sort of travel together, too, in that a workerType is scoped to a provisioner.  They could be displayed together (either hidden under "More.." or visible):

```
Provisioner / WorkerType
aws-provisioner-v1 / gecko-3-b-linux
```

I still feel like Tags (and, now that I notice it, Source) are kind of using a lot of space.  The tags could get mixed in with the existing "owner" pill, and Source could probably be mixed in there, too (just as a link -- no need to show the URL).

**Comment by helfi92** [comment]
> Provisioner is now behind "More.." but I still see "workerType".

I'll bring back the provisioner to where it was orignally. I kept the worker type displayed because it had a link to the worker-types view and I didn't want users to have to click an extra button to get to it.

> Source could probably be mixed in there, too (just as a link -- no need to show the URL).

I'll add `...` to make it fit in one line.

![screen shot 2018-11-06 at 9 44 18 am](https://user-images.githubusercontent.com/3766511/48071776-0f774f00-e1a9-11e8-9e2e-36df68cada5b.png)

> The tags could get mixed in with the existing "owner" pill

I'll give that a try.

**Comment by helfi92** [comment]
Closing as resolved. Please re-open if you are not satisfied with any of the changes.
